src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline

```diff
@@ -106,14 +106,14 @@ function LibraryServiceClient(gaxGrpc, grpcClients, opts) {
   var port = opts.port || DEFAULT_SERVICE_PORT;
   var sslCreds = opts.sslCreds || null;
   var clientConfig = opts.clientConfig || {};
-  var appName = opts.appName || 'gax';
-  var appVersion = opts.appVersion || gax.version;
 
   var googleApiClient = [
-    appName + '/' + appVersion,
+    'gl-node/' + process.versions.node,
+    'gccl/' + require('../../package.json').version,
```

Comment:
This should be optional parameters `lib_name` and `lib_version` since the wrapping library is what should define this header key and version. This is because we expect gapic to be wrapped by more than google cloud in the future.